opencti-platform/opencti-front/src/private/components/settings/sub_types/entitySetting/EntitySettingAttributeLine.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { FunctionComponent, useState } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import ListItemText from '@mui/material/ListItemText/ListItemText';
+import ListItem from '@mui/material/ListItem';
+import Skeleton from '@mui/material/Skeleton';
+import { ListItemButton, ListItemSecondaryAction } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { KeyboardArrowRightOutlined, MoreVertOutlined, ShortTextOutlined } from '@mui/icons-material';
+import Drawer from '@mui/material/Drawer';
+import { DataColumns } from '../../../../../components/list_lines';
+import { Theme } from '../../../../../components/Theme';
+import ErrorNotFound from '../../../../../components/ErrorNotFound';
+import EntitySettingAttributeEdition from './EntitySettingAttributeEdition';
+import { EntitySettingAttributeLine_attribute$key } from './__generated__/EntitySettingAttributeLine_attribute.graphql';
+import {
+  EntitySettingAttributes_entitySetting$data,
+} from './__generated__/EntitySettingAttributes_entitySetting.graphql';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  itemIcon: {
+    color: theme.palette.primary.main,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 5,
+  },
+  itemIconDisabled: {
+    color: theme.palette.grey?.[700],
+  },
+  drawerPaper: {
+    minHeight: '100vh',
+    width: '50%',
+    position: 'fixed',
+    overflow: 'auto',
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    padding: 0,
+  },
+  goIcon: {
+    position: 'absolute',
+    right: -10,
+  },
+}));
+
+const entitySettingAttributeLineFragment = graphql`
+  fragment EntitySettingAttributeLine_attribute on TypeAttribute {
+    name
+    type
+    label
+    multiple
+    mandatory
+    mandatoryType
+    defaultValues {
+      id
+      name
+    }
+    scale
+  }
+`;
+
+interface EntitySettingAttributeLineProps {
+  node?: EntitySettingAttributeLine_attribute$key
+  dataColumns?: DataColumns
+  entitySetting: EntitySettingAttributes_entitySetting$data
+}
+
+const EntitySettingAttributeLine: FunctionComponent<EntitySettingAttributeLineProps> = ({
+  node,
+  dataColumns,
+  entitySetting,
+}) => {
+  if (!node) {
```

Comment:
Can the node be updated by the parent component during its lifecycle ? If yes : this could break the UI